src/components/Schedule.js

```diff
@@ -39,81 +71,90 @@ const Schedule = () => {
+                                    </p>
+                                  </div>
+                                  <div className="schedule__location">
+                                    <p
+                                      className="schedule__location__title"
+                                      title={event.title}
+                                    >
+                                      {event.title}
+                                    </p>
+                                    <a
+                                      className="schedule__location__zoom"
+                                      href={event.location}
+                                      title={event.location}
+                                      // make the link open in a new tab
+                                      target="_blank"
+                                      rel="noopener noreferrer"
+                                    >
+                                      {event.location}
+                                    </a>
+                                  </div>
                                 </div>
-                              </td>
-                              <td className="schedule__audience">
-                                <p
-                                  className={
-                                    event.audience
-                                      ? "schedule__audience__type"
-                                      : ""
-                                  }
-                                >
-                                  {event.audience}
-                                </p>
-                                <p
-                                  className={
-                                    event.company
-                                      ? "schedule__audience__company"
-                                      : ""
-                                  }
-                                >
-                                  {event.company}
-                                </p>
-                              </td>
-                              <td className="schedule__calendar">
-                                <img src={CalendarIcon} alt="calendar icon" />
-                              </td>
-                            </tr>
-                          );
-                        })}
-                      </tbody>
-                    </Table>
+                                {width >= 1000 && (
+                                  <AdditionalAttributes event={event} />
+                                )}
+                              </div>
+                            </Card.Header>
+                            <Accordion.Collapse eventKey={idx.toString()}>
+                              <Card.Body>
+                                <strong>Description:&nbsp;</strong>
+                                {event.description}
+                                {width < 1000 && (
```

Comment:
is this here for small screens?

Reply:
yep. I used 1000 because both 960 and 1200 didn't work super great on this

Reply:
Right now technically we don't have the mobile version, so I am kinda free styling lol